modules/ref-dashboard-configuration-options.adoc

```diff
@@ -59,6 +59,18 @@ To show these items, set the value to `false`.
 
 LAB-tuning is a Technology Preview feature in this {productname-short} release.
 | `spec.dashboardConfig.` +
+`disableKueue` 
+
+(Technology Preview) | `true` | When enabled, new projects automatically receive a local queue, allowing administrators to select hardware profiles configured with a default queue name and priority in their workloads.
```

Comment:
Given how confusing these options are (enabled usually = false) to help make it more clear, change the wording to something like:
Set the value to `false` if you want new projects to automatically receive....